test/jdk/java/awt/print/PrinterJob/ImagePrinting/AlphaPrintingOffsets.java

```diff
@@ -0,0 +1,237 @@
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.BasicStroke;
+import java.awt.Color;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.awt.print.Book;
+import java.awt.print.PageFormat;
+import java.awt.print.Paper;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+import javax.print.attribute.HashPrintRequestAttributeSet;
+import javax.print.attribute.PrintRequestAttributeSet;
+import javax.print.attribute.standard.Sides;
+import javax.swing.JFrame;
+
+import static java.awt.print.PageFormat.LANDSCAPE;
+import static java.awt.print.PageFormat.PORTRAIT;
+import static java.awt.print.PageFormat.REVERSE_LANDSCAPE;
+
+/*
+ * @test
+ * @bug 8307246
+ * @key printer
+ * @library ../../../regtesthelpers
+ * @build PassFailJFrame
+ * @summary Test for comparing offsets of images drawn with opaque
+ *          and translucent colors printed in all orientations
+ * @run main/manual AlphaPrintingOffsets
+ */
+
+public class AlphaPrintingOffsets {
+    private static final String INSTRUCTIONS =
+            "Tested bug occurs only on-paper printing " +
+                    "so you mustn't use PDF printer\n\n" +
+                    "1. Java print dialog should appear.\n" +
+                    "2. Press the Print button on the Java Print dialog.\n" +
+                    "3. Please check the page margin rectangle are properly " +
+                    "drawn and visible on all sides on all pages.\n" +
+                    "If so, press PASS, else press FAIL.\n\n" +
+                    "Also you may run this test in paper-saving mode. Due " +
+                    "to tested bug affects pages printed with transparency " +
+                    "in LANDSCAPE and REVERSE_LANDSCAPE orientations there " +
+                    "is an option to print only 2 pages affected. " +
+                    "To do it pass PaperSavingMode parameter.";
+
+    private static boolean isPaperSavingMode = false;
+
+    public static void main(String[] args) throws Exception {
+        if (PrinterJob.lookupPrintServices().length > 0) {
+
+            String instructionsHeader = "This test prints 6 pages with page " +
+                    "margin rectangle and a text message. \n";
+            if (args.length > 0) {
+                isPaperSavingMode = args[0].equals("PaperSavingMode");
+            }
+            if (isPaperSavingMode) {
+                instructionsHeader = "This test prints 2 pages with page " +
+                        "margin rectangle and a text message. \n";
+            }
+            PassFailJFrame.builder().instructions(instructionsHeader + INSTRUCTIONS)
+                    .rows(15).testUI(() -> createTestUI()).build().awaitAndCheck();
```

Comment:
```suggestion
            PassFailJFrame.builder()
                    .instructions(instructionsHeader + INSTRUCTIONS)
                    .rows(15)
                    .testUI(() -> createTestUI())
                    .build()
                    .awaitAndCheck();
```
What do you think? Is it easier to read?